package.json

```diff
@@ -29,7 +25,7 @@
     "sauce-connect-launcher": "^1",
```

Comment:
If there's no more tests, I think you can cleanup some of the dependencies here and the `wct.conf.json` file

Reply:
Right... thanks.  I'll clean this up a bit more.  😄